getting-started-ng/success.md

```diff
@@ -1,7 +1,12 @@
-Now that you’ve learned the NativeScript basics, you’re ready to move on to our next tutorial, which has you built a complete iOS and Android app from scratch.
+Now that you’ve learned the NativeScript basics, you’re ready to move on to our next tutorial, which has you built a complete iOS and Android app from scratch. To continue you can:
 
-You can complete that tutorial in Playground, or, if you prefer to work in a local development environment, you can continue in [NativeScript Sidekick](https://www.nativescript.org/nativescript-sidekick) or the [NativeScript command-line interface](https://docs.nativescript.org/angular/tutorial/ng-chapter-1).
+* **Option 1. Complete the next tutorial in Playground**  
+Complete <a id="btn-next-tutorial" href="/?template=groceries-ng&tutorial=groceries-ng"><strong>Build a Full App</strong>&nbsp;tutorial in Playground</a>
 
-As always, if you get stuck, feel free to reach for help on [**Stack Overflow**](https://stackoverflow.com/questions/tagged/nativescript) or ask our experts in [**NativeScript community Slack channel**](http://developer.telerik.com/wp-login.php?action=slack-invitation). Have fun building with NativeScript!
+* **Option 2. Start working in a local environment with NativeScript CLI & Playground Preview App**  
+If you’re comfortable working with a command-line interface, you can install NativeScript CLI by running `npm i nativescript -g` and use `tns preview` command to preview you app on real device the same way you do it in Playground. 
```

Comment:
[Suggestion] Add "**the**" in front of NativeScript CLI ("(...) you can install **the** NativeScript CLI (...)"). Also, consider adding "**the**" in front of "`tns preview` command".